Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml

```diff
@@ -79,6 +79,53 @@
   name: tomato killer
   description: Looks like it's not you eating tomatoes today, it's the tomatoes eating you.
   components:
+  - type: GhostRole
+    prob: 0.05
+    makeSentient: true
+    allowSpeech: true
+    name: ghost-role-information-tomatokiller-name
+    description: ghost-role-information-tomatokiller-description
+    rules: ghost-role-information-familiar-rules
+    raffle:
+      settings: short
+  - type: GhostTakeoverAvailable
+  - type: CanEscapeInventory
+    baseResistTime: 3
+  - type: Puller
+    needsHands: false
```

Comment:
```suggestion
```
Ok, one last thing that came up. Please remove the ability for pulling, as other small mobs without hands like mice or slimes cannot do this either. Once that is done this should be good to merge.